### DISCUSSION TITLE
How to update label text based on option menu input

### DISCUSSION THREAD

**Discussion by DarkKnight263** [TomSchimansky/CustomTkinter#857] [Q&A] [answered]
Hi I'm very new to using this library, could someone guide me on how I could update the label based on option menu input, I have been made aware of trace in tkinter, how can that be implemented here?

**Comment by Akascape** [comment, marked as answer]
@DarkKnight263 
Example of changing label with option menu:
```python
import customtkinter
       
app = customtkinter.CTk()

def change_label(value):
    Label.configure(text="Option: "+value)
    
Label = customtkinter.CTkLabel(app, text="Option: 1")
Label.grid(pady=10)

Option_Menu = customtkinter.CTkOptionMenu(app, values=["1","2","3"], command=change_label)
Option_Menu.grid(pady=10, padx=10)

app.mainloop()
```

**Reply by DarkKnight263** [reply]
Thanks a lot man, .....also checked out ur work....damn,someday i wish to be as good as u
